src/components/PrimaryPanes/SourcesTreeItem.js

```diff
@@ -213,7 +213,7 @@ class SourceTreeItem extends Component<Props, State> {
       case "webpack://":
         return "Webpack";
       default:
-        return `${item.name}`;
```

Comment:
would we want to unescape this closer to the source? like in getURL or something?

Reply:
maybe we should add a jest test for this?

Reply:
This place is the only occurrence of the problem